gemset.nix

```diff
@@ -0,0 +1,400 @@
+{
+  addressable = {
+    dependencies = ["public_suffix"];
+    groups = ["default" "development" "test"];
+    platforms = [];
+    source = {
+      remotes = ["https://rubygems.org"];
+      sha256 = "022r3m9wdxljpbya69y2i3h9g3dhhfaqzidf95m6qjzms792jvgp";
+      type = "gem";
+    };
+    version = "2.8.0";
+  };
+  bip_mnemonic = {
+    groups = ["default"];
+    platforms = [];
+    source = {
+      remotes = ["https://rubygems.org"];
+      sha256 = "0p9kdsy0i6x4170l83726lrm52c1g62pfzi6ip3nx0ysj6h3qffy";
+      type = "gem";
+    };
+    version = "0.0.4";
+  };
+  capybara = {
+    dependencies = ["addressable" "mini_mime" "nokogiri" "rack" "rack-test" "regexp_parser" "xpath"];
+    groups = ["development" "test"];
+    platforms = [];
+    source = {
+      remotes = ["https://rubygems.org"];
+      sha256 = "0ji9kyb01dpnjbvpyb0c481cpnisd6wx6div6rywi9fihk66627w";
+      type = "gem";
+    };
+    version = "3.33.0";
+  };
+  cardano_wallet = {
+    dependencies = ["httparty"];
+    groups = ["default"];
+    platforms = [];
+    source = {
+      remotes = ["https://rubygems.org"];
+      sha256 = "1q7c2vhfwsbwm2arwgbhgpmx4q81fahya3pd9yfx2h1a8riaqygr";
+      type = "gem";
+    };
+    version = "0.3.21";
```

Comment:
Is there a way to regenerate this file from `Gemfile` in case some dependencies change?

Reply:
Right, the tool is called `bundix`